code/controllers/subsystem/minor_mapping.dm

```diff
@@ -42,15 +42,33 @@ SUBSYSTEM_DEF(minor_mapping)
 	var/turf_temperature = proposed_turf.temperature
 	return turf_gasmix.has_gas(/datum/gas/oxygen, 5) && turf_temperature < NPC_DEFAULT_MAX_TEMP && turf_temperature > NPC_DEFAULT_MIN_TEMP
 
-/datum/controller/subsystem/minor_mapping/proc/place_satchels(amount=10)
+/datum/controller/subsystem/minor_mapping/proc/place_satchels(amount=2)
+	///List of possible tiles to be placed above the satchels
+	var/list/possible_replacement_tiles = list( //We want these to look out of place most of the time
+		/turf/open/floor/iron = 1,
+		/turf/open/floor/iron/edge = 1,
+		/turf/open/floor/iron/half = 1,
+		/turf/open/floor/iron/corner = 1,
+		/turf/open/floor/iron/large = 1,
+		/turf/open/floor/iron/small = 1,
+		/turf/open/floor/iron/diagonal = 1,
+		/turf/open/floor/iron/herringbone = 1,
+		/turf/open/floor/plastic = 1,
```

Comment:
Could you show me two pictures of how these look like? I think these will just look _too_ out of place and just give some lucky people instant smugglers

Reply:
I'd rather see people finding out by pure accident or by hunting for them

Reply:
![image](https://github.com/tgstation/tgstation/assets/40921881/0e07eed1-326d-4cbc-92c2-6e32dc272d14)

![image](https://github.com/tgstation/tgstation/assets/40921881/42e3a0b4-3019-404d-8247-c0b07c695440)

Reply:
Two things of note:
I chose the plastic tile specifically to be the most out of place of the bunch
Not every place in the station is as iron-tile dependent as hallways and such, just most places, and so the tiles will look more out of place in places like that

Also the point of spawning a random tile above the satchel is so that some crewmembers get lucky and find one, rather than just those looking with a t-ray scanner, or those that happen to have one. I want the idea that, if you choose to go looking for it, your up against the entire crew, not just the other guy looking for it and some engineers.

Reply:
This feels really lame and looks worse, to be honest? I've always liked the idea of people needing to go out of their way to find them and never really being certain if it was spawned there or if someone hid it there. This makes a distinct difference between spawned and antag-bought ones.

Like I would rather have your original suggestion than this (which, if I was less charitable, I could argue is the _point_ of this change, but I won't).

Reply:
Yeah thats fair, and seems like a consistent problem people have that I can understand. Im just gonna remove it, im not exactly married to the idea anyway.